src/main/scala/de/bripkens/ha/Configuration.scala

```diff
@@ -25,7 +24,7 @@ object Configuration {
 
 @JsonCreator
 case class Configuration(@JsonProperty("endpoints") endpoints: Set[HealthCheckEndpoint],
-                         @JsonProperty("reporters") reporters: Map[String, AbstractReporterConfig],
+                         @JsonProperty("reporters") reporters: Map[String, ReporterConfig],
```

Comment:
This is probably out of the scope of this PR, but why not use circe or some advanced Scala based JSON lib instead of Jackson?